index.js

```diff
@@ -254,14 +254,16 @@ module.exports = {
   },
 
   _getExtensions(config) {
+    let shouldHandleTypeScript = this._shouldHandleTypeScript();
     let emberCLIBabelConfig = config['ember-cli-babel'] || {};
-    return emberCLIBabelConfig.extensions || ['js'];
+    return emberCLIBabelConfig.extensions || (shouldHandleTypeScript ? ['js', 'ts'] : ['js']);
```

Comment:
Any reason to not _always_ default to `['js','ts']`?

Reply:
*probably* no reason not to. If a user wanted to transpile ts some other way for some reason (or ignore ts or something) they could just manually configure the extensions and exclude ts.

Reply:
Ya, that's what I was thinking. Let's do that.

Reply:
So, turns out this *has* to be conditional. If a package (say, `ember-ajax`) contains `.ts` files and that package isn't using ember-cli-typescript >= 4.0 (say, `ember-ajax`) Babel will choke because it tries to transpile the `.ts` files, but the TypeScript transforms won't be added for Babel in that addon tree. We can only safely tell it to include `.ts` files when we know the Babel instance will have the TS transforms (`shouldHandleTypeScript === true`).

I will revert this back.